src/lib/github.ts

```diff
@@ -66,12 +66,27 @@ export const getRelease = (
   tag: string
 ): Promise<string> => {
   const client = gh.client(token, clientOptions);
-  return client
-    .release(name, `tags/${tag}`)
-    .infoAsync()
-    .then((release: Array<{[key: string]: string}>) => {
-      return release[0].tag_name;
-    });
+  return new Promise((resolve, reject) => {
+    client.get(
+      `/repos/${name}/tags`,
+      {per_page: 100},
+      (err: Error, code: number, resp: [{name: string}]) => {
+        if (err) {
+          return reject(err);
+        } else if (code !== 200)
+          return reject(Error(`unexpected http code = ${code}`));
```

Comment:
I don't know if just saying `Error()` is the same as `new Error()` but this don't look right.  Also, as written, this will hide the original stack trace.  Would rather prepend the message and re-reject the existing exception.